packages/superset-ui-demo/storybook/stories/plugins/plugin-chart-word-cloud/Stories.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { SuperChart } from '@superset-ui/chart';
+import { select } from '@storybook/addon-knobs';
+import WordCloudChartPlugin from '@superset-ui/plugin-chart-word-cloud';
+import LegacyWordCloudChartPlugin from '@superset-ui/plugin-chart-word-cloud/esm/legacy';
+import data from './data';
+
+new WordCloudChartPlugin().configure({ key: 'word-cloud2' }).register();
+new LegacyWordCloudChartPlugin().configure({ key: 'legacy-word-cloud2' }).register();
+
+export default {
+  title: 'Chart Plugins|plugin-chart-word-cloud',
```

Comment:
Can we change this to "Plugins" so "Core Packages" can always display on the top of the navigation in Storybook:


![image](https://user-images.githubusercontent.com/335541/80416111-70f11900-8888-11ea-9600-575205681b3e.png)